lib/netplay/netqueue.cpp

```diff
@@ -200,29 +190,22 @@ void NetQueue::writeRawData(const uint8_t *netData, size_t netLen)
 	while (buffer.size() - used > 1)
```

Comment:
Should probably be:
```suggestion
	while (buffer.size() - used >= NetMessage::HEADER_LENGTH)
```
Since that's the minimum valid message size (and we don't want to risk reading beyond the end of the buffer when fetching `len`).